include/div_ops.hxx

```diff
@@ -23,12 +23,12 @@
 
  */
 
-#ifndef __DIV_OPS_H__
-#define __DIV_OPS_H__
+#ifndef HERMES_DIV_OPS_H
```

Comment:
Hi @bendudson I had a google about what this means and I see that it's to prevent the header file from being included more than once across the code. But how come you're changing the name here? This is a merge conflict and I'm not sure how to resolve it.

Reply:
It's just to prevent the file from being included twice. The name doesn't really matter, but names starting `__` are not supposed to be used, since they may conflict with compiler internals.